src/content/esp3d/v3.x/hardware/esp32-s3/uicpal/_index.md

```diff
@@ -0,0 +1,55 @@
++++
+archetype = "section"
+title = "UICPAL ESP32 S3"
+description =  "UICPAL ESP32 S3"
+weight = 3
++++
+
+[AliExpress](https://aliexpress.com/item/1005006095863180.html)
+
+## Specs
+* ESP32-S3 with 16MB flash memory, 8MB PSRAM (qspi), 3D antenna, RGB led, Camera and SD card reader (SDIO 1bit)
+
+
+![image](front.png?width=400px)
+![image](schematic.png?width=400px)
+
+## Pinout
+
+SD:
+| Function | GPIO|
+| :-: | :-: |
+| CMD | 39  |
+| CLK | 42  | 
+| D0  | 41  |
+
+Camera: CAMERA_MODEL_UICPAL_ESP32S3
+
+RGB Led:  WS2812_PIN  33
+ 
+
+{{% notice style="orange" title="Warning" icon="exclamation" %}}
+The UART converter on this board requires DTR and RTS to be disabled in order to receive Serial output.
+
+In Platform.io, add the following lines:
+
+```
+monitor_dtr = 0
+monitor_rts = 0
+```
+
+Or if using arduino-cli, use the following arguments:
+
+```
+$ arduino-cli monitor --port COM5 -c baudrate=115200,rts=off,dtr=off
+```
+
+Arduino IDE does not support this board invidually and hacking the
+system files feels too much efforts. An easy workaround is to
```

Comment:
Regarding the Arduino IDE, there are several esp-s3 boards compatible with the default "ESP32-S3 Dev module" profile. In the case of the UICPAL, it would also work with that profile, except for the serial port. I tried for some time to change the board attributes for that profile to disable RTS and DTR. But every attempt I made would not be conclusive. That's when I made the mistake of switching to ESP Dev Module instead of S3, and it worked. That's when I noticed that the non-s3 profile already has the RTS and DTR disabled from the start .

Reply:
well I am not sure to follow :
with esp3d only use the ESP32S3 Dev Module
```
esp32s3.name=ESP32S3 Dev Module
esp32s3.vid.0=0x303a
esp32s3.pid.0=0x1001

esp32s3.bootloader.tool=esptool_py
esp32s3.bootloader.tool.default=esptool_py

esp32s3.upload.tool=esptool_py
esp32s3.upload.tool.default=esptool_py
esp32s3.upload.tool.network=esp_ota

esp32s3.upload.maximum_size=1310720
esp32s3.upload.maximum_data_size=327680
esp32s3.upload.flags=
esp32s3.upload.extra_flags=
esp32s3.upload.use_1200bps_touch=false
esp32s3.upload.wait_for_upload_port=false

esp32s3.serial.disableDTR=false
esp32s3.serial.disableRTS=false
```
https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L515-L534

the lines are already present

Reply:
RTS and DTR need to be disabled, hence
```
esp32s3.serial.disableDTR=true
esp32s3.serial.disableRTS=true
```

Reply:
I see the espressif syntax is upside down compare to arduino syntax, I have read too fast

monitor_dtr = 0
monitor_rts = 0

esp32s3.serial.disableDTR=true
esp32s3.serial.disableRTS=true

Reply:
so it would be more clean to ask user to change the lines in board.txt, I do not see a way to change the setting in ide

Reply:
> so it would be more clean to ask user to change the lines in board.txt, I do not see a way to change the setting in ide

That's what I tried "hacking system files" and failed to get conclusive results. boards.txt gets cached in temp folder and even after trying to delete it the results were not what I expected ( I saw my changes in menus but it would not affect the serial console).

Hence why temporarily switching to other profile is more comfortable.

Reply:
usually you need to update the boards.txt with IDE closed, then open IDE to load the boards.txt

Reply:
I tried many things. Including adding a menu entry like the official PR that added support for disableRTS/disableDTR suggested. The menu entries would show up but it would never take effect when testing serial output. Even a brand new profile just for UICPAL would not work. I don't know where that setting was still being stored/mirrored beside Roaming folder, but the only option that was working in my attempts was switching to an existing profile with RTS/DTR disabled from the get go.